File: async-scala-client/src/main/scala/io/swagger/client/model/PurchaseTransactionOut.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PurchaseTransactionOut (
  header: PurchaseHeaderOut,
lines: List[PurchaseLinesOut],
calculatedTaxSummary: PurchaseCalculatedTaxSummaryForService,
processingInfo: ProcessingInfo
)

File: javascript-closure-angular-client/API/Client/PurchaseTransactionOut.js
goog.provide('API.Client.PurchaseTransactionOut');

/**
 * @record
 */
API.Client.PurchaseTransactionOut = function() {}

/**
 * @type {!API.Client.PurchaseHeaderOut}
 * @export
 */
API.Client.PurchaseTransactionOut.prototype.header;

/**
 * @type {!Array<!API.Client.PurchaseLinesOut>}
 * @export
 */
API.Client.PurchaseTransactionOut.prototype.lines;

/**
 * @type {!API.Client.PurchaseCalculatedTaxSummaryForService}
 * @export
 */
API.Client.PurchaseTransactionOut.prototype.calculatedTaxSummary;

/**
 * @type {!API.Client.ProcessingInfo}
 * @export
 */
API.Client.PurchaseTransactionOut.prototype.processingInfo;

